extensions/linear/src/components/IssueActions/IssueActions.tsx

```diff
@@ -421,6 +450,8 @@ export default function IssueActions({
           onChange={setDueDate}
         />
 
+        <Action.PickDate title="Set Reminder" shortcut={{ modifiers: ["cmd"], key: "h" }} onChange={setReminder} />
```

Comment:
Finally, it'd be great to limit the dates you can select. For example, Linear reminders can't be set for past dates. We should maybe add a `min` and `max` prop to `Action.PickDate` (and `Form.DatePicker` as well.

Reply:
Yeah this popped up before, definitely something that we should add. Feels handy!